CONTRIBUTING.md

```diff
@@ -89,16 +89,21 @@ to make reviews and retrospection easy. Use your git commits to provide context
 for the reviewers, and the folks who will be reading the codebase in the months
 and years to come.
 
-Finalized commit messages should look similar to the following format:
+We're trying to keep all commits in `master` to follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) format.
+See [conventions](#commit-conventions) for more info on types and scopes.
+We are using squash and merge approach to PRs which means that commit descriptions are generated from PR descriptions.
```

Comment:
I am not really sure if the desc for the commit, that is created in master when a PR is merged is, is created from PR desc. Instead I have seen that commit desc just has the other commits/their desc from the PR.
I was looking into this PR for example https://github.com/kanisterio/kanister/pull/2664 below is how this PR is merged
![image](https://github.com/kanisterio/kanister/assets/19333642/d3a574dc-6570-4c99-991e-d530e853becd)

As we can see the commit message is PR title but the commit desc is the commits of the PR.

Reply:
I meant pr title. Updated.